packages/mux-player-react/src/lazy.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useEffect, useState } from 'react';
+
+import ConditionalSuspense from './ConditionalSuspense';
+import useIsBrowser from './useIsBrowser';
+import useIsIntersecting from './useIsIntersecting';
+
+import type { MuxPlayerProps, MuxPlayerRefAttributes } from './index';
+
+const MuxPlayerIndex = React.lazy(() => import('./index'));
+
+interface FallbackProps extends MuxPlayerProps {
+  onIntersection?: () => void;
+}
+const Fallback = (props: FallbackProps) => {
+  const { style, className, onIntersection, placeholder } = props;
+
+  const intersectionRef = React.useRef<HTMLElement>(null);
+  const isIntersecting = useIsIntersecting(intersectionRef);
+
+  useEffect(() => {
+    if (isIntersecting && onIntersection) {
+      onIntersection();
+    }
+  }, [isIntersecting, onIntersection]);
+
+  return (
+    <>
+      {/* 
+        Why do we have a mux-player element before the mux-player bundle is even loaded?
+        Before the bundle is loaded, this mux-player element just acts like a div. 
+        However, by calling this placeholder "mux-player", 
+        it now gets the same CSS applied to it that the eventual "real" mux-player element will.
+      */}
+      {/* TODO: can we add mux-player to JSX.IntrinsicElements */}
+      {/* @ts-ignore */}
+      <mux-player
+        ref={intersectionRef}
+        data-mux-player-react-lazy-placeholder
+        className={className || ''}
+        style={{
+          // default mux-player styling
+          aspectRatio: '16/9',
+          display: 'block',
+          backgroundColor: 'var(--media-control-background, #000)',
+          width: '100%',
+          // user-implemented styles
+          ...style,
+          // and styles that are important enough to override the user's preferences
+          position: 'relative',
+          backgroundImage: placeholder ? `url(${placeholder})` : undefined,
+          backgroundRepeat: 'no-repeat',
+          backgroundSize: 'contain',
+          '--controls': 'none',
+        }}
+        placeholder={placeholder}
+        // since there's a possibility that the bundle loads before Suspense clears this placeholder,
+        // we need to make sure that the placeholder doesn't get rendered and isn't interactive
+        nohotkeys
+        aria-hidden
+        tabIndex="-1"
+      >
+        {/* Overlay */}
+        <div
+          style={{
+            position: 'absolute',
+            inset: 0,
+            backgroundColor: 'var(--controls-backdrop-color, rgba(0, 0, 0, 0.6))',
```

Comment:
Is this the correct way to get the color of the controls backdrop? The placeholder/poster overlay?

Reply:
Yes, though, if you change it to `var(--controls-backdrop-color, var(--controls, rgba(0, 0, 0, 0.6)))`, it'll auto turn off if folks use `--controls: none` to turn off controls

Reply:
Ah shoot. I can't take this suggestion. The placeholder _always_ has `--controls: none` so that the player chrome wouldn't load on the placeholder before Suspense has had the chance to replace the placeholder with the real player. 

Since the placeholder assigns the `--controls` attribute, there's no way that I can think of of knowing the user's `--controls` preference within. So we'll just default to always darkening the placeholder.

Reply:
Ah, cool. Forgot that it's the placeholder and that controls are disabled.